gffquant/counters/count_matrix.py

```
@@ -0,0 +1,165 @@
+        counts[:, 5] = counts[:, 4] * combined_scaling_factor
+
+        self.counts = counts
+
+        return self
+    
+    def dump(self, state="genes", labels=None,):
+        with open(f"CountMatrix.{state}.txt", "wt") as _out:
+            if labels is None:
+                for index, counts in self:
+                    print(index, *counts, sep="\t", file=_out)
+                for (index, counts), label in zip(self, labels):
```

Ruff (0.8.2)

Loop control variable `index` not used within loop body

(B007)